src/main/java/org/reactome/release/cosmicupdate/Main.java

```diff
@@ -0,0 +1,227 @@
+import java.sql.SQLException;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.stream.Collectors;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.reactome.release.common.ReleaseStep;
+import org.reactome.release.common.dataretrieval.cosmic.COSMICFileRetriever;
+import org.reactome.util.general.GUnzipCallable;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+
+public class Main extends ReleaseStep
+{
+	@Parameter(names = {"-u"}, description = "Specifies that the updates should be performed.")
+	private boolean executeUpdate;
+	
+	@Parameter(names = {"-d"}, converter = DurationConverter.class,
+			description = "Download the files, if the age of local files exceeds the specified max age, or if the files don't exist."
+					+ "Omitting this parameter means download will not occur. Specifying -d with a 0 value will force a download."
+					+ "Format for a Duration can be found here: https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html#parse-java.lang.CharSequence-")
+	private Duration fileAge;
+	
+	private static String COSMICMutantExport;
+	private static String COSMICFusionExport;
+	private static String COSMICMutationTracking;
+
+	private static String COSMICMutantExportURL;
+	private static String COSMICFusionExportURL;
+	private static String COSMICMutationTrackingURL;
+	
+	private static String COSMICUsername;
+	private static String COSMICPassword;
+	
+	private static final Logger logger = LogManager.getLogger();
+	private static long personID;
+
+	public static void main(String... args)
+	{
+		try
+		{
+			try(Reader configReader = new FileReader("src/main/resources/config.properties"))
+			{
+				Properties props = new Properties();
+				props.load(configReader);
+				Main.COSMICMutantExport = props.getProperty("pathToMutantExportFile", "./CosmicMutantExport.tsv");
+				Main.COSMICFusionExport = props.getProperty("pathToFusionExportFile", "./CosmicFusionExport.tsv");
+				Main.COSMICMutationTracking = props.getProperty("pathToMutationTrackingFile", "./CosmicMutationTracking.tsv");
+				Main.COSMICMutantExportURL = props.getProperty("urlToMutantExportFile", "https://cancer.sanger.ac.uk/cosmic/file_download/GRCh38/cosmic/v92/CosmicMutantExport.tsv.gz");
+				Main.COSMICFusionExportURL = props.getProperty("urlToFusionExportFile", "https://cancer.sanger.ac.uk/cosmic/file_download/GRCh38/cosmic/v92/CosmicFusionExport.tsv.gz");
+				Main.COSMICMutationTrackingURL = props.getProperty("urlToMutationTrackingFile", "https://cancer.sanger.ac.uk/cosmic/file_download/GRCh38/cosmic/v92//CosmicMutationTracking.tsv.gz");
```

Comment:
Since this URL appears multiple times, could it be a global variable? Also related to https://github.com/reactome/release-cosmic-update/pull/1/files#r552245526

Reply:
Which URL appears multiple times?

Reply:
I should have said partial URL: `https://cancer.sanger.ac.uk/cosmic/file_download/GRCh38/cosmic/v92/`

Reply:
Ah ok. Yes, that part _could_ be extracted. Not sure I will...